integration/test-network/README.md

```diff
@@ -8,6 +8,13 @@ Make sure you have installed [yq](https://github.com/mikefarah/yq).
 Note that you will version v3.4.1 or larger. 
 For Ubuntu, `sudo snap install yq` is the easiest way to get a good version.
 
+Note if you run the FPC test network from within our FPC docker dev container, please
+use Ubuntu 20.04 Docker images by setting the corresponding settinngs in your `config.override.mk`
+before you build the dev container with `cd utils/dockder && make dev && make run`.
+```Makefile
+DOCKER_BUILD_OPTS=--build-arg UBUNTU_VERSION=20.04 --build-arg UBUNTU_NAME=focal
+```
```

Comment:
Thanks Michael, very good comments here! See my last fixup which addresses all your comments (except making Ubuntu 20.04 our new default).

Reply:
> * that said, as mentioned in #527, That said, should we maybe take this as an opportunity and make 20.04 the default (in docker and README) instead of 18.04? Contrary to 18.04, 20.04 is an LTS and probably most folks (like e.g., Jordan in his recent tests!) anyway start with that? That doesn't necessarily would force us to go everywhere to 20.04 in our test systems and would have the advantage that both setups would be tested (20.04 via CI) and 18.04 via our local tests.... (of course, we could also trivially add both OS to our git action via a strategy->matrix like we do for PDO for different interpreters ..). Admittedly, this might be something for a separate (though fairly small/straightforward) PR.  But a move to 20.04 is something which actually be good to add to 1.0 if we would do that anyway sooner than later?

Yes, I also think we should move to Ubuntu 20.04 as our new default in documentation and docker. I've created Issue #530 for that. I would not go the route adding tests for Ubuntu 18.04 to CI until we get community feedback that this is a strong requirement.